docs/user_guide/da_vinci_client/da_vinci_client.md

```diff
@@ -0,0 +1,10 @@
+---
+layout: default
+title: Da Vinci Client
+parent: User Guides
+has_children: true
+permalink: /docs/user_guide/da_vinci_client
```

Comment:
I'd like to suggest a different structure... We already have a `Write APIs` section under `User Guides`. I think I'd like to have a `Read APIs` page alongside it, and the `Da Vinci Client` page can be nested under that. The Record Transformer stuff can be a section within the DVC page itself, rather than nested as a separate page.

You already figured out the `grand_parent` stuff correctly, so that's good. The above restructuring would also need a `grand_parent` annotation on the deepest level (the DVC page).